components/PageControl/src/MDCPageControl.h

```diff
@@ -97,6 +97,18 @@
  */
 @property(nonatomic) BOOL defersCurrentPageDisplay;
 
+/**
+ When this value is set to YES, the indicators will ascend from right to left in an RTL environment.
+
+ @note In general, the MDCPageControl's UIScrollViewDelegate forwarding methods make assumptions
+ about the originating scrollview's page number based off its contentOffset. When this property is
+ set to YES in an RTL environment, a leftmost content offset will be considered the last page in the
+ scrollview, as opposed to the first.
+
+ The default value is NO.
```

Comment:
Can we assume that people will want RTL when they are in an RTL locale and drop this property?
If we absolutely need this property, can we make the default behavior to properly observe RTL?

Reply:
@ianegordon , what do you mean by "drop" this property? Like not use it? If a user is in an RTL locale and this property is set to `NO` the page control works the way it always has, i.e. if a scrollview is passed into one of the `UIScrollViewDelegate` forwarding methods, and it's `contentOffset` is `CGPointZero`, the page control will assume it's on the 0th page. If this property is set to `YES` in an RTL locale, the page control will interpret a `CGPointZero` `contentOffset` as the last page of the scrollview. Because there is probably a lot of code relating `contentOffset` to page number, if this property didn't exist and the page control always respected `effectiveUserInterfaceLayoutDirection`, a bunch of clients would break. Case in point, our `PageControlTypicalUseExample` example broke until I added this property, set it to `YES`, and added logic to translate `contentOffset` to page number in an RTL locale.

Reply:
To summarize, this is a migration flag. @andrewoverton do we intend to deprecate and remove this flag eventually?

Reply:
I'm not sure what the future of this flag is, or even the component. Earlier today Ian mentioned deprecating the whole component offline, so who knows 🤷‍♂️